src/main/groovy/wooga/gradle/unity/UnityPluginExtension.groovy

```diff
@@ -84,6 +90,20 @@ trait UnityPluginExtension implements UnitySpec,
         assetsDir.set(value)
     }
 
+    private final DirectoryProperty packagesDir = objects.directoryProperty()
+
+    /**
+     * @return The Assets directory, where all serialized assets by an Unity project are located
```

Comment:
The documentation string seems to be wrong.

Reply:
Ah, good catch. I will fix it.